app/bundles/CoreBundle/Tests/Form/RequestTraitTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * @copyright   2020 Mautic Contributors. All rights reserved
+ * @author      Mautic, Inc.
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Tests\Form;
+
+use Mautic\CoreBundle\Form\RequestTrait;
+use PHPUnit\Framework\Assert;
+use Symfony\Component\Form\Extension\Core\Type\ButtonType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Form;
+use Symfony\Component\Form\FormConfigBuilder;
+use Symfony\Component\Form\FormFactoryBuilder;
+
+class RequestTraitTest extends \PHPUnit\Framework\TestCase
+{
+    use RequestTrait;
+
+    /**
+     * @var Form
+     */
+    private $form;
+
+    protected function setUp(): void
+    {
+        $factoryBuilder = new FormFactoryBuilder();
+        $factoryBuilder->addType(new SubmitType());
+        $factoryBuilder->addType(new ButtonType());
+        $factory = $factoryBuilder->getFormFactory();
+
+        $dispatcher = $this->getMockBuilder(
+            'Symfony\Component\EventDispatcher\EventDispatcherInterface'
```

Comment:
Let's use constants over strings. Even better to create a use statement at the top of the file to make this shorter and more readable. Also there is shorter syntax if you'll use `createMock` method
```php
        $dispatcher = $this->createMock(EventDispatcherInterface::class);
```